src/Components/Body.js

```diff
@@ -42,6 +42,7 @@ export default function Body() {
               title={item.title}
               target="_blank"
               rel="noreferrer"
+              onKeyDown={onLinkAccess}
```

Comment:
Why do we need this? 
Wouldn't it be better to stick to the default behaviour of links?

Reply:
it doesn't hurt to handle 'space' for link, but not necessary. can be removed can be kept.